src/traffic_server/InkAPI.cc

```diff
@@ -4934,12 +4934,11 @@ TSHttpSsnClientVConnGet(TSHttpSsn ssnp)
 TSVConn
 TSHttpSsnServerVConnGet(TSHttpSsn ssnp)
 {
-  TSVConn vconn       = nullptr;
   PoolableSession *ss = reinterpret_cast<PoolableSession *>(ssnp);
   if (ss != nullptr) {
-    vconn = reinterpret_cast<TSVConn>(ss->get_netvc());
+    return reinterpret_cast<TSVConn>(ss->get_netvc());
```

Comment:
This is unnecessary for H2 to Origin, and inconsistent with a similar code in `TSHttpTxnServerVConnGet`.